extensions/wikia/ExactTargetUpdates/lib/exacttarget_soap_client.php

```diff
@@ -5,21 +5,14 @@ class ExactTargetSoapClient extends SoapClient {
 	public $username = NULL;
 	public $password = NULL;
 
-	/**
-	 * @param bool $retryAttempt Is this call a repeated attempt to send data. Needs to be available for warning
-	 * handler that decides whether to requeue request.
-	 */
-	function __doRequest($request, $location, $saction, $version, $one_way = 0, $retryAttempt = false) {
+	function __doRequest($request, $location, $saction, $version, $one_way = 0) {
 		$doc = new DOMDocument();
 		$doc->loadXML($request);
 
 		$objWSSE = new WSSESoap($doc);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this parameter "$one_way" to match the regular expression ^[a-z][a-zA-Z0-9]*$. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=php%3AS117)